e2e/pages/listing-creation.po.ts

```diff
@@ -2,6 +2,27 @@ import { ElementHelper } from '../utils/element-helper';
 import { element, by, browser, ExpectedConditions } from 'protractor';
 import { ListingCreationForm } from '../models/listing-creation-form';
 import { Wait } from '../enums/wait';
+import { Logger } from '../utils/logger';
+
+interface ListingNote {
+  selector: string,
+  parentList: string
+}
+
+export const LISTING_NOTES = {
+  SPECIAL_REQUIREMENTS: {
+    selector: 'note-SpecialRequirements',
+    parentList: 'noteList'
+  } as ListingNote,
```

Comment:
I don't wont to be picky, but `as` keyword cast object to given type, In this particular example it does not cause a big issue but if you are casting the object to other one stong typing stops working here.
The recommended way would be to use `:Type` approach. In this example you can create an interface and use it just here or make inline type definition like `{SPECIAL_REQUIREMENTS: ListingNote ...} `etc

Reply:
I didn't think it's that important in this case, but will do